code/modules/atmospherics/auxgm/breathing_classes.dm

```diff
@@ -23,7 +23,7 @@
 /datum/breathing_class/oxygen
 	gases = list(
 		GAS_O2 = 1,
-		GAS_PLUOXIUM = 8,
+		GAS_O3 = 2,
```

Comment:
potent respiratory hazard and pollutant near [ground level](https://en.wikipedia.org/wiki/Ground_level_ozone)